app/src/adapters/db/__init__.py

```diff
@@ -9,7 +9,7 @@
 Usage:
     import src.adapters.db as db
 
-    db_client = db.init()
+    db_client = db.init_postgres_client()
```

Comment:
now that we have separate clients i feel like we don't really need the `init` abstraction anymore, what do you think about just creating the client directly i.e.

```
from src.adapters.db.clients import PostgresDBClient

db_client = PostgresDBClient()
```